status-resolver/build.gradle

```diff
@@ -23,11 +24,22 @@ android {
     }
 }
 
+repositories {
```

Comment:
should not be here

Reply:
1625e84cb028ce7b65cc6bd34201660fe27780f8